txscript/pkscript.go

```diff
@@ -13,13 +13,21 @@ import (
 )
 
 const (
-	// pubKeyHashSigScriptLen is the length of a signature script attempting
-	// to spend a P2PKH script. The only other possible length value is 107
-	// bytes, due to the signature within it. This length is determined by
-	// the following:
-	//   0x47 or 0x48 (71 or 72 byte data push) | <71 or 72 byte sig> |
-	//   0x21 (33 byte data push) | <33 byte compressed pubkey>
-	pubKeyHashSigScriptLen = 106
+	// minPubKeyHashSigScriptLen is the minimum length of a signature script
+	// that spends a P2PKH output. The length is composed of the following:
+	//   Signature length (1 byte)
+	//   Signature without variable R and S values (73 - 33 - 33 bytes)
+	//   Public key length (1 byte)
+	//   Public key (33 byte)
+	minPubKeyHashSigScriptLen = 1 + (73 - 33 - 33) + 1 + 33
```

Comment:
`(73 - 33 - 33)`? So just `7`? That isn't a valid signature length.

Reply:
In place, I think you just want `71` here.

Reply:
I've seen cases where the R and/or S values of the signature end up being less than the usual 32 bytes, so the lower end of the range is meant to work around this.

Reply:
minimum signature length is 8: https://github.com/btcsuite/btcd/blob/master/btcec/signature.go#L91

Reply:
Ah, that's exactly what I was looking for!

Reply:
Fixed.